packages/amplication-git-service/src/utils/isInAmplicationIgnore.ts

```diff
@@ -0,0 +1,18 @@
+import minimatch from 'minimatch';
+
+export function isInAmplicationIgnore(
+  amplicationIgnoreStatements: string[],
```

Comment:
In glob language, individual lines are called `Expressions` and not `Statements`. Let's rename this to be in line with it please.

https://mincong.io/2019/04/16/glob-expression-understanding/#:~:text=In%20computer%20programming%2C%20glob%20patterns,function%2C%20glob()%20later%20on.

Reply:
You are right, I'm used to the AST statements vs expressions...